arangod/Cluster/RestAgencyCallbacksHandler.cpp

```diff
@@ -61,16 +63,21 @@ RestStatus RestAgencyCallbacksHandler::execute() {
                   "invalid JSON");
     return RestStatus::DONE;
   }
-
+  
   try {
-    std::stringstream ss(suffixes.at(0));
-    uint32_t index;
-    ss >> index;
-
-    auto callback = _agencyCallbackRegistry->getCallback(index);
+    uint32_t index = basics::StringUtils::uint32(suffixes.at(0));
+    auto cb = _agencyCallbackRegistry->getCallback(index);
```

Comment:
Please talk with Michael.  I believe he renames "cb" to "callback" whenever he can.  Here you are renaming "callback" to "cb".  Maybe let @jsteemann be the referee while you two fight it out.